compiler/optimizations/gpuTransforms.cpp

```diff
@@ -733,11 +744,102 @@ bool GpuizableLoop::extractUpperBound() {
   return true;
 }
 
-void GpuizableLoop::reportNotGpuizable(const BaseAST* ast, const char *msg) {
+static bool shouldSkipInStaticTrace(CallExpr* call) {
```

Comment:
`shouldSkipInStaticTrace`? DYM `shouldSkipInStackTrace` or am I misunderstanding something?

Reply:
I think a stacktrace is typically a runtime trace (how the function got from A to B), whereas here we're printing a "static" trace of how the function was found -- which may not be a valid runtime trace.